.gitignore

```
@@ -390,3 +390,8 @@ gcov.css
 # https://clangd.llvm.org/ cache folder
 .clangd/
 .cache/
+
+# ninja tool
+.ninja
+build.ninja
+run_ninja_env.bat
```

Add a missing newline at the end of file (as it was before):

```suggestion
run_ninja_env.bat

```